rcon/automods/automod.py

```diff
@@ -231,11 +231,16 @@ def on_connected(rcon: Rcon, _, name: str, steam_id_64: str):
         logger.debug("No automod is enabled")
         return
 
+    # get detailed player info for use by on_connected_hook
+    detailed_player_info: GetDetailedPlayer = rcon.get_detailed_player_info(name)
```

Comment:
This will fail on anyone with a whitespace or pineapple name and not all mods require it (like weapon enforcement in seeding).

I think we want to try/catch this so people with bugged names don't cause the rest to fail?

Reply:
Good, um, catch. I will take a look at it maybe returning None if it fails.